rust/kcl-lib/src/std/mirror.rs

```diff
@@ -70,56 +65,81 @@ async fn inner_mirror_2d(
                     }),
                 )
                 .await?;
+
+            if let OkWebSocketResponseData::Modeling {
+                modeling_response: OkModelingCmdResponse::EntityMirror(mirror_info),
+            } = &resp
+            {
+                let face_edge_info = &mirror_info.entity_face_edge_ids;
+
+                let _ = starting_sketches
+                    .iter_mut()
+                    .zip(face_edge_info.iter())
+                    .try_for_each(|(sketch, info)| {
+                        sketch.id = info.object_id;
+                        let first_edge = info.edges.first().copied();
+                        match first_edge {
+                            Some(edge) => sketch.mirror = Some(edge),
+                            None => {
+                                return Err(KclError::new_engine(KclErrorDetails::new(
+                                    "No edges found in mirror info".to_string(),
+                                    vec![args.source_range],
+                                )))
+                            }
+                        }
+                        Ok(())
+                    });
```

Comment:
```suggestion
                    })?;
```

I'm pretty sure that without the `?`, it's just assigning the `Result` to `_` (in `let _ =`) and throwing it away. Then you should be able to delete `let _ =`.